### ISSUE TITLE
Source for raspberry pi half of the code

### ISSUE THREAD

**Issue by elmateo487** [samapraku/WifiDirectApp#1]
Hey,

Thanks for posting this. Do you have a repo you can share that has raspberry pi code for accepting the wifi direct connection?

Thanks

**Comment by samapraku** [comment]
Hello,

The connection handling on the raspberry was done using terminal commands. 

Basically, I used this post on SO to setup the raspberry for the Wi-Fi direct. The relevant part is up to the Reboot step in the post. 

https://raspberrypi.stackexchange.com/questions/94171/setting-up-wifi-direct-wifi-p2p-and-dhcp-server/94251#94251

After follwoing the guide in the post to set up the raspberry pi for Wi-Fi direct connection, I used the commands below to handle the connections on the device.

To accept a connection, I used the wps_pbc to accept the connection from android. It works like a WPS button on a Wi-Fi. Run the command below within seconds after initiating the connection from the android phone to accept the connection on the raspberry pi.  wps_pin option is also available if you want to use a pin instead.

1. Start Wi-Fi Direct on raspberry pi (after setting up Wi-Fi connection). This  adds the raspberry pi to the P2P group and puts it in listening mode for connections.

`sudo wpa_cli -iwlan0 p2p_group_add`

2. Accept incoming connections from Android device. ( Run few seconds after initiating connection from android)

`sudo wpa_cli -ip2p-wlan0-0 wps_pbc`
